src/imitation/data/types.py

```diff
@@ -52,7 +52,12 @@ class Trajectory:
     """Actions, shape (trajectory_len, ) + action_shape."""
 
     infos: Optional[np.ndarray]
-    """An array of info dicts, length trajectory_len."""
+    """An array of info dicts, shape (trajectory_len, ).
+
+    The info dict is returned by some environments `step()` and contains auxiliary
```

Comment:
Useful that you added it here, but this is not visible in the tutorial, so maybe tell the user to check these docs at least?